tasks/readme.rake

```diff
@@ -148,12 +148,35 @@ task :readme do
     contents
   end
 
+  def convert_to_list_items(elements)
+    elements.flat_map do |elem|
+      level = elem.value.options[:level]
+      text = elem.value.options[:raw_text]
+      id = elem.attr[:id]
+      padding = "  " * (level - 2)
+
+      ["#{padding}- [#{text}](##{id})"] + convert_to_list_items(elem.children)
+    end
+  end
+
+  def print_toc(contents)
+    require "kramdown"
+    section = "TOC"
+
+    doc = Kramdown::Document.new(contents, auto_id: true, toc_levels: (2..6))
```

Comment:
Is it equivalent to using https://kramdown.gettalong.org/rdoc/Kramdown/Converter/Toc.html?

Reply:
The next line is equivalent to that. What you've linked to is a converter which works on a `Document` instance, which is this.

Reply:
Ahh `to_toc`, thanks!